### PR TITLE
operator: always use software UUID as default machine name

### LINKED ISSUE #339: operator: use a software generated UUID as the default machine name
There are out there hw vendor that do not provide a unique SMBIOS SystemInformation/UUID value.
We should ensure that by default the machines (and the associated MachineInventories) get a unique name.

### PR DESCRIPTION
**Pull request by fgiudici**
The default machine name is in the format: m-{UUID}.
The UUID is generated via software if SMBIOS data is disabled, otherwise the SMBIOS {System Information/UUID} is used.
Since some hardware vendors don't properly fill the UUID SMBIOS data, let always provide a machine name based on a software generated UUID, to ensure name uniqueness by default.

Fixes #339

### CODE REVIEW

**Review by rdoxenham — APPROVED** [review]
Don't think I have write access, but LGTM!

**Review by davidcassany — APPROVED** [review]
LGTM, thanks :+1:

**Pull request merged**
